src/containers/Header/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { Logo } from '../../components'
+
+const Header = () => (
+  <header className='App-header'>
```

Comment:
Move the css to `src/containers/Header/styles.css`
and use header tag to apply css (delete className)

Reply:
done